memo_app.rb

```diff
@@ -2,9 +2,48 @@
 
 require 'sinatra'
 require 'json'
+require 'pg'
+require 'dotenv'
+
+Dotenv.load
+
+# MemoClass for Run SQL
+class Memo
+  class << self
+    def connection
+      PG.connect(
```

Comment:
```suggestion
      @connection ||= PG.connect(
```

のようにしてconnectionオブジェクトをキャッシュすると効率が良くなりそうです。

Reply:
ありがとうございます。
最初に呼ばれた際にconnectionオブジェクトを代入して、その後nullになっていない限り使い回せるということですよね。
下記コミットで取り込ませていただきました。
https://github.com/azitama6/memo-app/pull/2/commits/c0adc394f4a9566956842814616d2b01bef55231